src/tools/devtools/release-notes/release-notes-2.21.1-src.md

```diff
@@ -33,7 +33,7 @@ Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
 * Add a link to the DevTools [CONTRIBUTING](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md) guide to the About menu, and fixed the Discord link - [#4926](https://github.com/flutter/devtools/pull/4926)
 * Fix conflicting colors in light theme - [5067](https://github.com/flutter/devtools/pull/5067)
 
-## Changelog
+## Full commit history
+More details about changes and fixes are available from the
+[DevTools git log.](https://github.com/flutter/devtools/commits/master).
```

Comment:
**issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice). Use site variable.
```suggestion
To learn more about DevTools changes and fixes, check out the
[commit history]({{site.github}}/flutter/devtools/commits/master).
```